package-lock.json

```diff
@@ -0,0 +1,1875 @@
+{
```

Comment:
I'm indifferent if we want to include it or not. It does not get published, or affect users (afaik), but is for our benefit as the developers of this library. More info on this here: https://docs.npmjs.com/files/package-lock.json - let me know if you catch anything I didn't.